gt-scheduler/website
# Nathan/UI changes

### Summary

Resolves #313

- Moved github button according to design on Figma
- Added donation button
- Added donation banner component which is shown only once ever
- Replaced action button in information modal with donation button as per design

### How to Test

Run website and observe the new donate now button in the information modal, and the donate banner across the top. The new banner closes when you press the 'X', the GitHub button is in the bottom now, and the donate button is in the top navbar.
